src/v3/src/transformer/terminal/transformTerminalTransaction.ts

```diff
@@ -208,6 +209,14 @@ export const transformTerminalTransaction = (
 
   transformTerminalMessages(transaction, formBag);
 
+  createIdentifierContainer({
+    transaction,
+    widgetProps,
+    step: '',
+    setMessage: () => {},
+    isClientTransaction: false,
+  })(formBag);
```

Comment:
In addition to calling `createIdentifierContainer` custom transformer as part of the generic `uischema` transformer, we must also call it in the terminal transformer since some terminal states render `IdentifierContainer`